### ISSUE TITLE
[Talent] [Rouge] Talent "Vile Poisons" increases poison base damage instead of total

### ISSUE THREAD

**Issue by Tamaxofalgalon** [dalaranwow/dalaran-wow#5835]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

The Talent Vile Poisons currently increases poisons **base damage** by 7% for every rank instead of increasing the **total damage** dealt after Attack Power scaling.

Current Formula - Assuming 3/3 Vile Poisons (20% damage Modifier) // Instant Poison IX
`1.2 x 300 + 0.1 x Attack Power`

![Poison Scaling Proof](https://i.imgur.com/1GNJMJU.png)

_Current Value: 1.2 x 300 + 0.1 x 4745_ = **834**

**Expected behaviour**: Tell us what should happen instead

Vile Poisons should increase **total poison damage** dealt by up to 20% depending on rank.

Damage formula assuming 3/3 Vile Poisons (20% damage Modifier) // Instant Poison IX
`1.2 x ((300-400) + 0.1 x Attack Power)`

Damage formula assuming 3/3 Vile Poisons (20% damage Modifier) // Deadly Poison IX
`1.2 x (74 x 4 + 0.12 x Attack Power)`

![Damage Modifier](https://i.imgur.com/Ejvnx2y.png)

Expected values for the tooltip:
-Instant Poison: 1.2 x (300 + 0.1 x 4745) = **929**
-Deadly Poison: 1.2 x (74 x 4 +0.12 x 4745) = **1038**

This should be a fast and easy to implementfix as all of the formulas can be found in this ticket! (see "proof" and "expected behaviour" section)

**Steps to reproduce the problem**:

1.  Calculate current poison damage with the "current behaviour" formula
2.  Calculate expected poison damage with the "expected behaviour" formula
3.  Compare the outcomes to the ingame tooltip.

**Include proofs for this behaviour**

![Vile Poisons](https://i.imgur.com/MdQRX5q.png)
_Screen shot taken of a Wotlk database_
-->**Add Percent Modifier (Damage)**
-->**Add Percent Modifier (Damage over Time)**

![IP Formula](https://i.imgur.com/ePSCbmf.png)
_Instant Poison Formula taken from a Wotlk database_

![IP Formula2](https://i.imgur.com/2nUsxn0.png)
_Instant Poison Formula taken from WoWWiki_

![IP Formula](https://i.imgur.com/0tYIp1d.png)
_Instant Poison Formula taken from WoWWiki (1/3 Vile Poisons - 7% Damage Modifier)_
_Proof for Vile Poisons being a "total damage" multiplier_

![DPFormula](https://i.imgur.com/45QjQG3.png)
_Deadly Poison Formula taken from a Wotlk database_

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Vile Poisons: **16515**
Instant Poison IX: **43231**
Deadly Poison IX: **43233**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Tamaxofalgalon** [comment]
Dear developers, any updates on implementing the fixfor this issue?
